wafer/schedule/tests/test_views.py

```diff
@@ -1491,6 +1492,13 @@ def setUp(self):
         for index, item in enumerate(items):
             item.slots.add(slots[index // 2])
 
+        user = get_user_model().objects.create_user('john', 'best@wafer.test',
+                                                    'johnpassword')
+        talk = Talk.objects.create(title="Test talk", status=ACCEPTED,
+                                   corresponding_author_id=user.id)
+        talk_item = ScheduleItem.objects.create(venue=venue1, talk_id=talk.pk)
+        talk_item.slots.add(slots[4])
+
```

Comment:
We have this logic scattered across a few different schedule tests now. We really should unify that at some point - probably by using the helpers we have for the talk tests.

Reply:
I'll take a shot at it in a follow-up.